### PR TITLE
Better pull request templates

### PR DESCRIPTION
**Pull request by filips123**
This add different pull request templates for features and bugs.
Developers could now provide more details about PRs using some guidelines.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -12,3 +12,6 @@
 
   Learn more about contributing: https://www.gatsbyjs.org/docs/how-to-contribute/
 -->
+
+<!-- For fixing bugs use https://github.com/gatsbyjs/gatsby/compare/?template=bugs.md -->
```
Comment (yeskunall) [inline .github/PULL_REQUEST_TEMPLATE.md]: Feel like this will be missed by a lot of people 😅

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -12,3 +12,6 @@
...
+<!-- For pull requesting new features, improvements and changes use https://github.com/gatsbyjs/gatsby/compare/?template=features.md -->
```
Comment (yeskunall) [inline .github/PULL_REQUEST_TEMPLATE.md]: Same with this one.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (filips123) [reply .github/PULL_REQUEST_TEMPLATE.md]: I know... 
How can I make it more visible?

Comment (filips123) [reply .github/PULL_REQUEST_TEMPLATE.md]: Is it possible to have multiple pull request templates with select menu like for issues?

**Comment by m-allanson** [comment]
Thanks @filips123 👍 

Doing a bit of research on this, it seems like GitHub doesn't provide a way for someone to choose a 'type' of pull request, so these templates probably won't be seen by most people. 

I'm going to close this - for these templates to be useful there needs to be an easier way to view them, which GitHub doesn't provide at the moment. Maybe we can revisit this if GitHub changes how they handle PR templates?

However, if there's information that we should ask for _every_ PR, it would make sense to add that to the PR template. e.g. maybe it's useful to add _Description_ and _Related issue_ headings to the default template?

Would you like to open an issue to discuss making changes to the default PR template?

**Comment by filips123** [comment]
I created issue #10470 for that.

I can edit my PR to make them more visible, maybe using header or what do you think it will be the best, and you can then merge it. Unfortunately, I can't make more thing until GitHub supports that.

**Pull request closed without merging**